README.rst

```diff
@@ -1401,6 +1401,19 @@ And then configure RSpec in this way:
 This way, when running in recording mode, only the tests that have been marked with the
 ``:show_in_doc`` metadata will be run, and hence only those will be used as examples.
 
+If you want to distinguish your recorded examples from one another, you can provide a ``doc_title``
```

Comment:
It should be mentioned that this works only with RSpec. The ultimate patch would be to make it work also with minitest, in case https://github.com/ordinaryzelig/minispec-metadata/blob/master/lib/minispec-metadata/it.rb#L30 is present.